python/tests/integration_tests/test_runs.py

```diff
@@ -72,13 +72,15 @@ def my_chain_run(text: str):
 
     my_chain_run("foo", langsmith_extra=dict(project_name=project_name))
     executor.shutdown(wait=True)
-    for _ in range(5):
+    for _ in range(15):
         try:
             runs = list(langchain_client.list_runs(project_name=project_name))
             assert len(runs) == 3
             break
         except (ls_utils.LangSmithError, AssertionError):
             time.sleep(1)
+    else:
+        raise AssertionError("Failed to get runs after 5 attempts.")
```

Comment:
```suggestion
        raise AssertionError("Failed to get runs after 15 attempts.")
```

I need to fix this logic up heh thanks for the patience